test/langtools/jdk/javadoc/doclet/testAnnotationTypes/TestAnnotationTypes.java

```diff
@@ -24,7 +24,7 @@
 /*
  * @test
  * @bug      4973609 8015249 8025633 8026567 6469561 8071982 8162363 8182765 8223364
-             8242056 8261976 8223358
+             8242056 8261976 8223358 8313204
```

Comment:
That bug id differs from that of this PR; typo?

Reply:
Ditto reply.